### PR TITLE
fix: Return Node.js errors in callbacks

### PR DESCRIPTION
**Pull request by okdistribute**
This was causing an error upstream in Mapeo Desktop and Mapeo Core which expected errors to be passed through as Node.js errors. Without this. `error.toString()` returns `[Object object]` rather than the error message.

### CODE REVIEW

**Comment by noffle** [comment]
By the way, I think you can ignore the Travis failures. It was [failing in the same way](https://travis-ci.org/github/digidem/indexed-tarball/builds/655112660) before your PR.

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
-- '8'
-- '10'
+- '12.14.1'
```
[the remark this answers is not in the document]
Comment (gmaclennan) [reply .travis.yml:3]: I think it is worth testing all the modules that we depend on for Mapeo with the specific versions of Node that we are shipping with the apps.

On mobile, this is currently `v10.13.0` but we will shortly upgrade to `v12.16.3`. On Desktop we are using Electron `v6.1.7` which runs main process code with Node `v12.4.0`.

I think it is useful to test these specific versions because it will catch any use of specific features which were introduced through subsequent minor releases, or catch us hitting any bugs that are specific to a version of Node.

TL;DR I think the versions we test on travis should be `v10.13.0`, `v12.4.0`, `v12.16.3` and `v12` (e.g. latest).

[the remark this answers is not in the document]
Comment (okdistribute) [reply .travis.yml:3]: I'm updating electron to 12.14.1 which is why I added that here, so perhaps we should have 10.13.0, 12.4.0, and 12.14.1.

**Comment by noffle** [comment]
@okdistribute @gmaclennan This sounds reasonable.

**Comment by okdistribute** [comment]
@noffle the package already has a package-lock.json in it, so I don't want to take it away in this PR which is just a bug fix.

**Comment by okdistribute** [comment]
see #11
